WDAC-Policy-Wizard/app/src/MainForm.cs

```diff
@@ -1510,29 +1519,26 @@ public bool ConvertPolicyToBinary()
             Pipeline pipeline = runspace.CreatePipeline();
 
             // If multi-policy format, use the {PolicyGUID}.cip format as defined in https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/deploy-multiple-windows-defender-application-control-policies#deploying-multiple-policies-locally
-            string binaryFileName; 
-            if(this.Policy._Format == WDAC_Policy.Format.MultiPolicy)
+            string binaryFileName = string.Empty;
+            SiPolicy finalSiPolicy = Helper.DeserializeXMLtoPolicy(this.Policy.SchemaPath);
```

Comment:
Serialize Technically, Deserializing would be from bytes to xml

Reply:
Is this true? Summary for System.Xml.Serialization implies the opposite:

Summary:
        //     Deserializes the XML document contained by the specified System.IO.TextReader.
        //
        // Parameters:
        //   textReader:
        //     The System.IO.TextReader that contains the XML document to deserialize.
        //
        // Returns:
        //     The System.Object being deserialized.

Reply:
https://en.wikipedia.org/wiki/Serialization

> The opposite operation, **extracting a data structure from a series of bytes**, is deserialization, (also called unserialization or [unmarshalling](https://en.wikipedia.org/wiki/Unmarshalling)).